lighthouse-core/lib/dependency-graph/simulator/simulator.js

```diff
@@ -364,3 +364,8 @@ module.exports = Simulator;
  * @property {number} [layoutTaskMultiplier]
  */
 
+/**
+ * @typedef SimulationResult
```

Comment:
I discovered they're referenceable when you `require` in the file as I was getting the simulator stuff type checked, that seems like a decent way to prevent LH externs from ballooning with the more local typedefs, wdyt?

```js
const Simulator = require('../../simulator')

/**
 * @param {Simulator.SimulationResult} result
 */

Reply:
gah, I swear I checked this with `Audit` and it didn't work for jsdoc typedefs. Maybe it was fixed in 2.8? I can't remember if I tested before or after we updated. Or it was just user error :)

But since it works, agree that that's good for this kind of typedef